attributes/default.rb

```diff
@@ -1,11 +1,15 @@
 # encoding: UTF-8
 
 # Versions
-default['nsq']['version'] = '0.3.0'
-default['nsq']['go_version'] = 'go1.3.3'
+default['nsq']['version'] = '1.2.1'
```

Comment:
We have some of our topology already moved over to 1.2.1, so it'll be good to have any new nodes we build pick this up by default.